tests/unit/graph_reader/test_artifact_collection.py

```diff
@@ -64,26 +65,20 @@
         ),
     ],
 )
-def test_one_session(execute, input_script, artifact_list, expected_output):
+def test_one_session(
+    linea_db, execute, input_script, artifact_list, expected_output
+):
     """
     Test code refactor
     """
 
     code = pathlib.Path(
         "tests/unit/graph_reader/inputs/" + input_script
     ).read_text()
-    res = execute(code, snapshot=False)
+    execute(code, snapshot=False)
 
-    artifact_string = ", ".join([f'"{x}"' for x in artifact_list])
-    code = (
-        "from lineapy.graph_reader.artifact_collection import ArtifactCollection\n"
-        + "from lineapy.execution.context import get_context\n"
-        + "db = get_context().executor.db\n"
-        + f"ac = ArtifactCollection(db, [{artifact_string}])"
-    )
-    res = execute(code, snapshot=False)
-    ac = res.values["ac"]
-    refactor_result = ac.generate_module_text()
+    artifact_collection = ArtifactCollection(linea_db, artifact_list)
+    refactor_result = artifact_collection.generate_module_text()
```

Comment:
@lionsardesai: This change addresses LIN-561 (Declutter tests related to `ArtifactCollection`) as we discussed. Not sure if it results in speed improvement, but it's definitely less verbose code.

Reply:
This code became outdated as a result of rebase. Closing this and opening another thread for the equivalent code part.